examples/bcc/bash_readline/bash_readline.go

```diff
@@ -53,7 +54,28 @@ type readlineEvent struct {
 	Str [80]byte
 }
 
+var (
+	name string
+	h bool
+)
+
+func init() {
+	flag.BoolVar(&h, "h", false, "Print this help message")
```

Comment:
The flag package does print help text by default for `-h` / `--help`, so that's not needed I think.